docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/shape/squeeze-1.rst

```diff
@@ -17,7 +17,10 @@ Squeeze
 **Detailed description**: *Squeeze* can be used with or without the second input tensor.
 
 * If only the first input is provided, every dimension that is equal to 1 will be removed from it.
-* With the second input provided, each value is an index of a dimension from the first tensor that is to be removed. Specified dimension has to be equal to 1, otherwise an error will be raised. Dimension indices can be specified directly, or by negative indices (counting dimensions from the end).
+* With the second input provided, each value is an index of a dimension from the first tensor that is to be removed. Specified dimension should be equal to 1, otherwise it will be ignored and copied as is.
+Dimension indices can be specified directly, or by negative indices (counting dimensions from the end).
+
+Note: Updated behavior since 2024.3, request of squeezing dimension not equal to 1 is expected to be ignored instead of causing an error.
```

Comment:
If we're going to follow the convention of updating operators within the same opset whenever possible, maybe we should add a standardized `Changelog` header to affected ops' docs? It may be too early for that, but we can keep it in mind for the future.

Reply:
Thank you @slyalin for your comment and approval.
I've added the "Note" about updated behavior. 
Let me create a follow up PR to clarify it more, and discuss the spec changes separately (if needed).